decidim-core/app/types/decidim/core/user_entity_input_filter.rb

```diff
@@ -27,55 +27,46 @@ class UserEntityInputFilter < BaseInputFilter
       ```
       "
 
-      argument :type,
-               type: String,
-               description: "Filters by type of entity (User or UserGroup)",
-               required: false,
-               prepare: ->(value, _ctx) do
-                 type = value.downcase.camelcase
-                 type = "UserGroup" if %w(Group Usergroup).include?(type)
-                 { type: "Decidim::#{type}" }
-               end
+      argument :type, type: String, description: "Filters by type of entity (User or UserGroup)", required: false, prepare: ->(value, _ctx) do
```

Comment:
There's something weird with this file and identations, can you leave them as they used to be? One element per line makes for a better readability, IMO!

Reply:
I have tried to fix the indentation, however `bundle exec rubocop -a` has its own way ...